content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: blog
+title: 'Kubernetes 1.33: Fine-grained SupplementalGroups control graduates to beta'
+date: XXXX-XX-XX
+slug: fine-grained-supplementalgroups-control-beta
+author: >
+  Shingo Omura (LY Corporation)
+
+---
+
+Kubernetes 1.31 introduced new field `SupplementalGroupsPolicy` in Pod's Security Context to improve supplemental groups handling in containers in Pods.
+
+In Kubernetes 1.33, the feature graduates to beta and the corresponding feature gate (`SupplementalGroupsPolicy`) gets enabled by default.
+
+Please be aware that this beta release contains some behavioral breaking change. See [The Behavioral Changes Introduced In Beta](#the-behavioral-changes-introduced-in-beta) and [Upgrade Considerations](#upgrade-consideration) sections for details.
```

Comment:
do you have the same sections in documentation? It is valuable to have them in docs, not only in a blog post

Reply:
Basically yes. I added the behaviral changes as a note:

https://github.com/kubernetes/website/blob/7a9e33eb515a1afc2005fdef2127a701ef3548bb/content/en/docs/tasks/configure-pod-container/security-context.md?plain=1#L339-L356

But, I did not include "upgrade consideration" part in the doc. Would you think we should include it in the doc, too??